### DISCUSSION TITLE
HA Integration (3.73.9.20240130) lässt keine Konfigurtaionsänderung der Geräte zu

### DISCUSSION THREAD

**Discussion by Marco4223** [OpenCCU/OpenCCU#2681] [🇩🇪 Fragen&Antworten]
### Describe the issue you are experiencing

Moin,
nach langerer Zeit und einigen Updates wollte ich heute einen Taster (HM-LC-Dim1TPBU-FM) einmal richtig herum einbauen und in dem Zuge auch gleich einmal die Konfiguration korregieren. Gesagt getan, schalten gedreht und im HA -> RaspberryMatic das verhalten des Tasters geändert. 
Leider bleibt der Screen dan hängen und ich kann nurnoch die Seite neu laden. Es werden aber keine änderungen vorgenommen. 
Ich habe dieses nun bei mehreren Geräte getestet und bleibe immer an der Scree of Death ;) hängen.
![grafik](https://github.com/jens-maus/RaspberryMatic/assets/10371265/4b1338da-957d-463a-88a1-cabe62cc6fa0)

Ist dieses bekannt? Oder habe ich bei den letzten Updates etwas überlesen? 
VG
MarcO

### Describe the behavior you expected

Rückkehr zum normal Screen und geänderte Settings/Parameter

### Steps to reproduce the issue

Einstellungen 
Geräte
Gerät (z.B. ) finden und auf Einstellungen klicken
Eistellungen ändern...z.B. Taster Oben -> aus/an
Ok oder Übernehemen klicken

### What is the version this bug report is based on?

3.73.9.20240130

### Which base platform are you running?

ha-addon (HomeAssistant Add-on)

### Which HomeMatic/homematicIP radio module are you using?

RPI-RF-MOD

### Anything in the logs that might be useful for us?

```txt
And where can I get them in HA-Addon? Please help
```

**Comment by Marco4223** [comment]
Wenn ich im Bereich DevConfig auf einem Interface etwas ändern mchte und hier über HomeMatic Configuration -> 
[Device List] in den Einstellungen [NEQ1801906:1 -> NEQ1801906:1 -> Edit] auf [Submit Changes] klicke bekomme ich einen 404 Seite angezeigt. 
Hier die URL: https://[Meine homeassistant DNS Name]/api/hassio_ingress/K1zIqbCi4CAgZ-SAfuTLZeMZRvYa8YaTQvbshjp9tKw/api/hassio_ingress/K1zIqbCi4CAgZ-SAfuTLZeMZRvYa8YaTQvbshjp9tKw/tools/devconfig.cgi?cmd=link_paramset&iface=BidCos-RF&sender=NEQ1801906:1&receiver=NEQ1801906:1&show_reverse=1&sid=@KcyD2K7gO1@

**Comment by jens-maus** [comment]
Da dies kein allgemeiner Bug sein kann (läuft auf tausenden anderen Installationen problemlos) hab ich dein Issue mal in ein Diskussionsbeitrag konvertiert. Vielleicht können so andere auch entsprechend versuchen zu helfen.

**Reply by Marco4223** [reply]
Vielleicht eine Idee wo ich weiter absetzen kann?

**Reply by jens-maus** [reply]
Ich würde vmtl. erst einmal in der HA-Addon Config des RaspberryMatic Addons bei den Portfreigaben den Port 80 auf sowas wie 8080 setzen und dann die RaspberryMatic WebUI versuchen direkt via Port 8080 zu erreichen um ein Problem mit ingress auszuschließen.

**Reply by Marco4223** [reply]
Danke für den Tipp. Probiere ich nachher aus. 🤗

**Reply by Marco4223** [reply]
Naja, der 404 verschwindet..das aufhängen und Konfigurieren über die "normale" oberfläche hängt sich aber weiterhin auf. 
Hoffen wir mal das andere helfen können.

**Reply by jens-maus** [reply]
> Naja, der 404 verschwindet..das aufhängen und Konfigurieren über die "normale" oberfläche hängt sich aber weiterhin auf. 

Das beruhigt mich, denn dann liegt der Fehler nicht in RaspberryMatic ;) starte die Browser Konsole und such nach Fehlern bzw. Schau auf der Zentrale nach cgimail.* Dateien unter /usr/local/tmp und schau da rein...

**Reply by Marco4223** [reply]
Würde ich gerne...allerdings weis ich leider nicht wie ich das heir mache "starte die Browser Konsole" (oder ich verstehe nicht was damit gemeint ist) und ich bin ich am grübeln wie ich an die Files kommen soll. SSH? Oder gobts da noch was anderes?

**Reply by Baxxy13** [reply]
Browserkonsole = F11 drücken
SSH Zugang in der WebUI freischalten, in der AddOn - Config den SSH Port freigeben , auf 2022 z.B. 
Verbinden dann mit PuTTy, WinSCP o.ä.

**Comment by Marco4223** [comment]
Gerade das Update v3.75.6.20240316 angezeigt kommt und alles funktioniert wieder. Fein fein fein
@Baxxy13 Danke für die Tipps. Bei F11 geht der Screen bei mir in den Vollbild Modus. Dann meint ihr F12? Die Entwickler/DevConsole?

**Reply by Baxxy13** [reply]
In der Tat war F12 gemeint.
Aber gut das es nun wieder funktioniert.
